### DISCUSSION TITLE
Support for COLRv1

### DISCUSSION THREAD

**Discussion by nitin2953** [character-map-uwp/Character-Map-UWP#197] [Ideas]
support for importing and/or exporting

**Comment by JohnnyWestlake** [comment]
WOFF2 is unlikely to happen for a while as DirectWrite team didn't want to support WOFF/WOFF2 (and WOFF is only supported as the WOFF font format is thankfully almost the same as a OTF so I perform manual conversion - which is not the case for WOFF2), COLRv1 will be contingent on DirectWrite adding support for it, which I believe can be requested at https://github.com/microsoft/WindowsAppSDK


It's probably possible too to request WOFF2 support for DirectWriteCore given the eventual cross-platform requirements, and the fact that someone, somewhere in Microsoft has code that handles this as Edge does it.

**Comment by JohnnyWestlake** [comment]
woff2 support has been added to the may-23 branch and will be pushed to the store branch soon

**Comment by JohnnyWestlake** [comment]
WOFF2 support is now in. COLRV1 isn't supported by any version of DirectWrite so won't be included for now.

**Comment by JohnnyWestlake** [comment]
COLRv1 support is currently in the dev branch for Character Map UWP, only for Windows 11 builds > 25905 (as this is when Microsoft has added COLRv1 support).

Currently it works in Type Ramp view and Quick Compare due to XAML not yet supporting COLRv1 on these builds.
